ansible/roles/ocp-workload-migration/README.adoc

```diff
@@ -0,0 +1,71 @@
+= Red Hat Cluster Migration Workload
+
+== Overview
+
+This is a workload for Red Hat Cluster Migration Tool.
```

Comment:
s/Red Hat Cluster Migration Tool/Cluster Application Migration Tool/

At the moment, the name is Cluster Application Migration Tool for our project.